entrypoint.sh

```diff
@@ -14,8 +14,9 @@ replace_in_files() {
         VALUE=`printenv "$VAR_NAME"`
 
         # Replace the variable only if it starts with the name of the directive and remove optional ';'
+        # Some directives will not have spaces after the directive name. i.e opcache
         find $PHP_CONFIG_FILES -type f -exec \
-            sed -i "s/^\(;\)\{0,1\}$DIRECTIVE = [^\n]\+/$DIRECTIVE = $VALUE/g" {} \;
+            sed -i "s/^\(;\)\{0,1\}$DIRECTIVE\( \)\{0,1\}=.*\+/$DIRECTIVE=$VALUE/g" {} \;
```

Comment:
Zero or one occurrence can be more succinctly written using the question mark:

```suggestion
            sed -i "s/^\(;\)\{0,1\}$DIRECTIVE\( \)?=.*\+/$DIRECTIVE=$VALUE/g" {} \;
```